docs/user/integration/sample-app/README.md

```diff
@@ -0,0 +1,122 @@
+# Integrate Custom Instrumentation App
+
+## Overview
+
+| Category| |
+| - | - |
+| Signal types | traces, metrics, logs |
+| Backend type | custom in-cluster, third-party remote |
+| OTLP-native | yes |
+
+Learn how to instrument your custom Golang application using the [OTel SDK](https://opentelemetry.io/docs/languages/) and exporting metrics and trace data using the OTel SDK. The sample application will be configured to push trace and metric data using OTLP to the collector that's provided by Kyma, so that they are collected together with the related Istio trace data.

```

Comment:
```suggestion
Learn how to instrument your custom Golang application using the [OTel SDK](https://opentelemetry.io/docs/languages/) and exporting metrics and trace data using the OTel SDK. You learn to configure the sample application to push trace and metric data using OTLP to the collector that's provided by Kyma, so that they are collected together with the related Istio trace data.

```